packages/log/manifest.yml

```
@@ -63,7 +63,7 @@ policy_templates:
       - name: custom
         title: Custom configurations
         description: >
-          Here YAML configuration options can be used to be added to your configuration. Be careful using this as it might break your configuration file.
+          Additional settings to be added to the configuration. Be careful using this as it might break the input as those settings are not validated and can override the settings specified above. See [`filestream` input settings docs](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-filestream.html) for details.
```

The Custom Logs integration uses the [log input](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html)
```suggestion
          Additional settings to be added to the configuration. Be careful using this as it might break the input as those settings are not validated and can override the settings specified above. See [`log` input settings docs](https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-input-log.html) for details.
```

---

🤦 indeed